sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -396,7 +397,9 @@ def query_items(
         if max_integrated_cache_staleness_in_ms:
             validate_cache_staleness_value(max_integrated_cache_staleness_in_ms)
             feed_options["maxIntegratedCacheStaleness"] = max_integrated_cache_staleness_in_ms
-
+        correlated_activity_id = GenerateGuidId()
+        if correlated_activity_id is not None:
```

Comment:
why are you doing this check? you create the variable right before, there should be no need to